packages/styles/scss/globals/_fonts.scss

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/type/scss/type';
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
```

Comment:
I realize y'all had a conversation in New York to go in this direction, but is it necessary to have this file when we could just point adopters to Carbon to do it themselves?

https://www.npmjs.com/package/@carbon/type

Reply:
And just remove the font files on our side? @photodow

Reply:
Yeah, so remove the imports, remove this new file, and just in our documentation somewhere point people to the link above.

Reply:
Fine with me either way, let's get some more feedback and see how everyone else feels

Reply:
> I realize y'all had a conversation in New York to go in this direction, but is it necessary to have this file when we could just point adopters to Carbon to do it themselves?

I don't have a problem with this at all.